src/apps/tenderize/ethereum/tenderize.swaptokens.token-fetcher.ts

```diff
@@ -0,0 +1,74 @@
+import { Inject } from '@nestjs/common';
+
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { buildDollarDisplayItem } from '~app-toolkit/helpers/presentation/display-item.present';
+import { ContractType } from '~position/contract.interface';
+import { PositionFetcher } from '~position/position-fetcher.interface';
+import { AppTokenPosition } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+
+import { TenderizeContractFactory } from '../contracts';
+import { ethereumEndpoint } from '../helpers/constants';
+import { ConfigResponse, configQuery } from '../helpers/queries';
+import { TENDERIZE_DEFINITION } from '../tenderize.definition';
+
+const appId = TENDERIZE_DEFINITION.id;
+const groupId = TENDERIZE_DEFINITION.groups.swaptokens.id;
```

Comment:
Rename group to `swapTokens`, and rename the file to `tenderize.swap-tokens.token-fetcher.ts`